src/app/frontend/common/components/resourcelist/event/template.html

```diff
@@ -62,14 +62,20 @@
 
       <ng-container [matColumnDef]="getDisplayColumns()[5]">
         <mat-header-cell *matHeaderCellDef>First Seen</mat-header-cell>
-        <mat-cell *matCellDef="let event">
+        <mat-cell *matCellDef="let event"
+          matTooltip="{{event.firstSeen | date:'medium'}}"
+          [matTooltipPosition]="top"
```

Comment:
This should be `matTooltipPosition="above"`

Enclosing the attribute in [] tries binding the value within the angular component and that's not needed.

"top" is not a valid value for this either, it's "above" -- See https://material.angular.io/components/tooltip/overview

Reply:
will fix and reverse thanks